app/uploaders/achievement_badge_uploader.rb

```diff
@@ -0,0 +1,13 @@
+# Image uploader for Achievement Badge
+# This class contains achievement badge specific settings for the image uploader.
+class AchievementBadgeUploader < ImageUploader
```

Comment:
You're saying that each kind of image requires their own image size restrictions?